### PR TITLE
Feature 47(Backend) error handling and validation

### PR DESCRIPTION
**Pull request by Ngha-Boris**
Two scripts were written into the project folder, one for error handling and the other for validation. They are named "error_handling.py" and "validation.py" respectively.

### CODE REVIEW

**Review by stephane-segning — CHANGES_REQUESTED** [review]
I cannot approve this PR, if the related dependencies (#46 ) are not done

**Comment by Ngha-Boris** [comment]
Hi sir @stephane-segning  i just updated my error handling code and also the ticket (#46) has been approved. Sir please review the work i pushed i will be waiting for you response. Thank you

**Pull request merged**
